defaults/main.yml

```diff
@@ -1 +1,33 @@
 ---
+
+# The database icinga2 will export it's configuration to.
+# default is "icinga2"
+icinga2_web_database_name: "icinga2"
```

Comment:
Remove the double ticks unless they're actually required to parse it into a string (e.g. when working with booleans) or YML syntax requires it.